corehq/apps/reports/standard/monitoring.py

```diff
@@ -1610,11 +1610,13 @@ def _rows_by_group(self, report_data):
             if group_name == 'no_group':
                 continue
 
-            case_sharing_groups = set(reduce(operator.add, [u['group_ids'] + [u['location_id']] for u in users], []))
-            active_cases = sum([int(report_data.active_cases_by_owner.get(u["user_id"].lower(), 0)) for u in users]) + \
-                sum([int(report_data.active_cases_by_owner.get(g_id, 0)) for g_id in case_sharing_groups])
-            total_cases = sum([int(report_data.total_cases_by_owner.get(u["user_id"].lower(), 0)) for u in users]) + \
-                sum([int(report_data.total_cases_by_owner.get(g_id, 0)) for g_id in case_sharing_groups])
+            owner_ids_nested = [
+                [user['user_id'].lower(), user['location_id']] + user['group_ids']
+                for user in users
+            ]
+            owner_ids = set(reduce(operator.add, owner_ids_nested, []))
```

Comment:
nit: `itertools.chain` would also be a good option here

Reply:
I removed this later on